<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>871</width>
      <height>731</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>书籍信息</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>781</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addButton">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>380</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>添加记录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="updateButton">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>430</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>提交修改</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteButton">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>490</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>删除选中行</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>321</width>
        <height>381</height>
       </rect>
      </property>
      <property name="title">
       <string>查询</string>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>书名</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>作者</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="booknameEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="authorEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="searchButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>300</y>
         <width>85</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>开始查询</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>出版社</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string> 类别</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string> 价格</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>出版年份</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="publisherEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>120</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="categoryEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>160</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lowPriceEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>200</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="earlyYearEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>240</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>210</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string> 到</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>250</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string> 到</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="highPriceEdit">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>200</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lateYearEdit">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>240</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="exitSearchButton">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>300</y>
         <width>101</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>退出查询</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="addFromFile">
      <property name="geometry">
       <rect>
        <x>572</x>
        <y>551</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>批量添加</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>用户信息</string>
     </attribute>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>831</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addButton2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>320</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>添加记录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteButton2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>460</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>删除选中行</string>
      </property>
     </widget>
     <widget class="QPushButton" name="updateButton2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>390</y>
        <width>85</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>提交修改</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>321</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>借书卡查询</string>
      </property>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>院系名</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>110</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>姓名</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_9">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_10">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>100</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="searchButton_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>160</y>
         <width>85</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>开始查询</string>
       </property>
      </widget>
      <widget class="QPushButton" name="exitSearchButton_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>160</y>
         <width>91</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>退出查询</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
